fastats/core/decorator.py

```diff
@@ -29,11 +29,7 @@ def fs(func):
 
     @wraps(func)
     def fs_wrapper(*args, **kwargs):
-        debug = kwargs.get('debug')
-        return_callable = kwargs.get('return_callable')
-
-        with suppress(KeyError):
-            del kwargs['return_callable']
```

Comment:
I see no reason for these shenanigans, did I miss anything?

Reply:
Yea, we need to keep debug and return_callable until we have a better option. I liked your idea of having an optional context manager like `with substitute(x) as func:`, which would take care of most of the use cases of return_callable

Reply:
OK - keeping it as is for now.